rich/console.py

```diff
@@ -758,7 +758,7 @@ def __init__(
         self._is_alt_screen = False
 
     def __repr__(self) -> str:
-        return f"<console width={self.width} {str(self._color_system)}>"
+        return f"<console width={self.width} {self._color_system}>"
```

Comment:
```suggestion
        return f"<console width={self.width} {self._color_system!s}>"
```
Oops, `format(num, "")` is the int for an IntEnum.